### ISSUE TITLE
Cannot use custom server emojis in setNavigationEmojis()

### ISSUE THREAD

**Issue by MaheshNat** [gazmull/discord-paginationembed#49]
When attempting to use custom emojis using setNavigationEmojis(), they are not captured by _awaitReponse(), and doesn't work

**Comment by secret0010** [comment]
Got the same issue, waiting on a solution.

**Comment by MaheshNat** [comment]
currently came up with a mock solution that probably doesn't work with some cases: https://hasteb.in/oxatokuc.kotlin. It's a replacement for the _awaitResponse() function in index.js

**Comment by d-shaun** [comment]
Works for me ™️.
Could you maybe share the code or at least the error message?

```
.setNavigationEmojis({
  delete: "😭",
  back: "🤠",
  jump: "😎",
  forward: "🙁",
});
```
![image](https://user-images.githubusercontent.com/59339807/101119558-95009f80-35b9-11eb-9153-18808d6368f1.png)

**Comment by MaheshNat** [comment]
The emojis you are using are standard emojis, I'm talking about guild-specific emojis; emojis that users have uploaded to a particular discord server and are not part of the standard emojis that discord provides you with.

**Comment by d-shaun** [comment]
Still works. 
```
.setNavigationEmojis({
  delete: "😭",
  back: "615469011563446272",
  jump: "😎",
  forward: "🙁",
})
```
![image](https://user-images.githubusercontent.com/59339807/101121569-73ee7d80-35be-11eb-8ad2-ce0482506688.png)
Like I asked before, do you get any errors?

**Comment by d-shaun** [comment]
OH! By "doesn't work", did you mean it's in the wrong order?

**Comment by gazmull** [comment]
Since there is no further discussion about this issue recently, I assume this is already resolved.
